<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialCommsForm</class>
 <widget class="QWidget" name="SerialCommsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data Bits</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbDataBits</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cmbDataBits"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Parity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbParity</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="cmbParity"/>
   </item>
   <item row="1" column="7">
    <widget class="QComboBox" name="cmbStopBits"/>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Stop Bits</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbStopBits</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>COM Port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbSerialList</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QSpinBox" name="spinBoxSplitChar">
     <property name="prefix">
      <string>Char</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>13</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QComboBox" name="cmbSerialList"/>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lblSplitChar">
     <property name="text">
      <string>Split At</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxSplitChar</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Baud Rate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>cmbBaud</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmbBaud"/>
   </item>
   <item row="2" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionCommSettingsChange">
   <property name="text">
    <string>CommSettingsChange</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmbBaud</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionCommSettingsChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbDataBits</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionCommSettingsChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbParity</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionCommSettingsChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbStopBits</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionCommSettingsChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
